INSTALL.md

```diff
@@ -108,6 +108,17 @@ _Tip:_
 composer require --update-no-dev "ezsystems/legacy-bridge:^2.0"
 ```
 
+### Recommended: Add addtional Legacy <=> eZ Platform integrations
+
+**[netgen/ezplatformsearch](https://github.com/netgen/ezplatformsearch)**
+
+When using either Solr or _(future)_ Elastic search bundle in eZ Platform, this eZ Publish extenion takes care of integrating with eZ Platform Search API, and among other things makes sure updates in legacy triggers search indexing updates, and all out of the box search featurs in eZ Publihs uses the same search index.
```

Comment:
`extenion` => `extension`
`triggers` => `trigger`

Also, how about `This also makes sure eZ Platform and eZ Publish Legacy use the same search index.` ?